source/documentation/deploying_apps/deploying_static_sites.md

```diff
@@ -1,3 +1,3 @@
 ## Deploy a static HTML page
 
-Refer to the [Deploy a test static HTML section](get_started.html#deploy-a-test-static-html-page) of the documentation. 
+Refer to the [Deploy a test static HTML section](get_started.html#deploy-a-test-static-html-page-against-the-machine) of the documentation. 
```

Comment:
@jpluscplusm or this one...

Reply:
@henrytk ^^ comment that might not have pinged you